tests/visual/components/AppSidebar/AppSidebar.visual.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * @copyright Copyright (c) 2020 Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ *
+ * @author Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import { mount } from 'cypress-vue-unit-test'
+
+// import Vue from 'vue'
+import AppSidebar from '../../../../src/components/AppSidebar/AppSidebar.vue'
+import ActionButton from '../../../../src/components/ActionButton/ActionButton.vue'
+
+// Server CSS styles
+import server from '../../../../styleguide/assets/server.css'
+import icons from '../../../../styleguide/assets/icons.css'
+import variables from '../../../../styleguide/assets/variables.css'
+
+// Import font so CI has the same
+import 'fontsource-roboto'
```

Comment:
I think we should find something later on to simplify this whole thing.
A `bootsrap.js` script  to import that does all this?

Reply:
Ideally, we anyway should be independent of the server styles, I think. Would simplify visual testing and the styleguide build. At the moment we actually need to update the server styles with every release (although it was never done) and cannot be sure that it looks the same on every Nextcloud server version.

Reply:
> Ideally, we anyway should be independent of the server styles, I think

I agree! But for fonts there isn't much choices :)
We could have our own base styling though, like variables.scss or icons.
That way we have a *standard* across our components to build their styles, that should help us and avoid having to write lots of unneeded small lines :wink: